jmaqs-utilities/src/test/java/com/magenic/jmaqs/utilities/helper/PropertyManagerUnitTest.java

```diff
@@ -0,0 +1,30 @@
+package com.magenic.jmaqs.utilities.helper;
+
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNull;
+
+public class PropertyManagerUnitTest {
+    /**
+     * An invalid key for the property manager to invoke failure
+     */
+    private String INVALID_KEY = "does-not-exist";
+
+    /**
+     * The default value provided to the property manager
+     */
+    private String DEFAULT_VALUE = "default";
+
```

Comment:
Can you add a test for a successful get of a property?  While you would have to set the property yourself, it would at least allow us to know if something breaks from Oracle's quickened release cycles.

Reply:
test added